src/python/pants/core/goals/package.py

```diff
@@ -187,9 +188,19 @@ async def package_asset(workspace: Workspace, dist_dir: DistDir) -> Package:
 
 @dataclass(frozen=True)
 class TraverseIfNotPackageTarget(ShouldTraverseDepsPredicate):
+    """This predicate stops dep traversal at any package targets.
```

Comment:
Hmm let's say "after package targets." so its unambiguous if the package target is included.